<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLayerDialog</class>
 <widget class="QDialog" name="CreateLayerDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layerNameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="layerTypeBox">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="numericTypeLayout">
        <item>
         <widget class="QRadioButton" name="mNumericTypeButton">
          <property name="text">
           <string>&amp;Numeric Layer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="numericTypeComboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Int 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UInt 8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Int 16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UInt 16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Int 32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UInt 32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Float 16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Float 32</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DatabaseTypeLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="mDatabaseTypeButton">
          <property name="text">
           <string>Database La&amp;yer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layerOptionLayout">
     <item>
      <layout class="QHBoxLayout" name="resolutionLayout">
       <item>
        <widget class="QLabel" name="resolutionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="resolutionComboBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>2048x2048</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4096x4096</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCreate</sender>
   <signal>released()</signal>
   <receiver>CreateLayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>released()</signal>
   <receiver>CreateLayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNumericTypeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>numericTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
